docs/reference/docker.md

```
@@ -1,26 +1,164 @@
+daemon was already running, add your user to the `docker` group, then log out
+and in for the change to be effective.
+
+Solution 2: use rootless docker as described above; the way to set this up
+depends on your distribution and in some cases is a matter of installing a
+variant package like `docker-rootless`.
+
+### Docker daemon errors initializing graphdriver
+
+Solution: the most likely reason is that you are using a filesystem like `zfs`
+or `btrfs` which requires a special storage driver, click
+[link](https://docs.docker.com/engine/storage/drivers/select-storage-driver/)
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)